models/balancer/arbitrum/balancer_arbitrum_trades.sql

```diff
@@ -0,0 +1,86 @@
+{{ config(
+    schema = 'balancer_arbitrum',
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "balancer",
+                                \'["bizzyvinci"]\') }}'
+    )
+}}
+
+
+with v2 as (
+    select
+        '2' as version,
+        tokenOut as token_bought_address,
+        amountOut as token_bought_amount_raw,
+        tokenIn as token_sold_address,
+        amountIn as token_sold_amount_raw,
+        poolAddress as project_contract_address,
+        s.evt_block_time,
+        s.evt_tx_hash,
+        s.evt_index
+    from {{ source('balancer_v2_arbitrum', 'Vault_evt_Swap') }} s
+    inner join {{ source('balancer_v2_arbitrum', 'Vault_evt_PoolRegistered') }} p
+    on s.poolId = p.poolId
+    {% if is_incremental() %}
+        where s.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+),
+prices as (
+    select * from {{ source('prices', 'usd') }}
+    where blockchain = 'arbitrum'
+    {% if is_incremental() %}
+        and minute >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+)
```

Comment:
Please add `project_start_date` for these model, as we can apply default incremental filters to `prices.usd` and `*.transactions` models.

Reply:
you mean I should add `if not is_incremental` statements like [this](https://github.com/duneanalytics/spellbook/blob/main/models/uniswap/ethereum/uniswap_v2_ethereum_trades.sql#L109-L111)?

Reply:
yes, exactly

Reply:
I need help with the test. It failed to run initial model without details

Reply:
I will rerun the dbt test and try to find the cause.

Reply:
[Initial models passed](https://github.com/duneanalytics/spellbook/actions/runs/4084743658/jobs/7055549350#step:9:29).